packages/aws-cdk-lib/aws-elasticloadbalancingv2-actions/lib/cognito-action.ts

```diff
@@ -65,6 +67,18 @@ export interface AuthenticateCognitoActionProps {
    * @default Duration.days(7)
    */
   readonly sessionTimeout?: Duration;
+
+  /**
+   * Allow HTTPS outbound traffic to communicate with the IdP.
+   *
+   * Set this property to false if the IP address used for the IdP endpoint is identifiable
+   * and you want to control outbound traffic.
+   * Then allow HTTPS outbound traffic to the IdP's IP address using the listener's `connections` property.
+   *
+   * @default true
+   * @see https://repost.aws/knowledge-center/elb-configure-authentication-alb
+   */
+  readonly allowHttpsOutbound?: boolean;
```

Comment:
I added a property that allows customers to control whether to allow HTTPS outbound traffic. This makes it possible to meet the requirements of customers who don't want to allow outbound traffic.